docs/docs/building-from-source.md

```diff
@@ -5,15 +5,15 @@ Users are recommended to use the [pre-built xemu binaries](https://github.com/mb
 ## Windows
 
 Windows builds are cross-compiled from Ubuntu. If you would like to build *on* Windows, you can use WSL2 and Docker. See [official Docker
-documentation](https://docs.docker.com/docker-for-windows/wsl/) for how to get WSL2 and Docker set up.
+documentation](https://docs.docker.com/docker-for-windows/wsl/) for how to get WSL2 and Docker set up. In the instructions below for windows, It can be used in Powershell/Command prompt but is recommended to do this in your wsl2 terminal on windows.
 
 ```bash
+# Configure git NOTE: Change back To true after use if needed
+git config --global core.autocrlf false
+
 # Clone and build
-git clone https://github.com/mborgerson/xemu.git
-docker run --rm -v $PWD/xemu:/xemu -w /xemu \
-    -e CCACHE_DIR=/xemu/ccache \
-    mborgerson/xemu-ubuntu-win64-cross:latest \
-    ./build.sh -p win64-cross
+git clone --recurse-submodules https://github.com/mborgerson/xemu.git
```

Comment:
Nit: --recurse-submodules shouldn't be needed here

Reply:
On powershell when doing --recurse-submodules for cloning. It allowed it to build to the end.

Reply:
This shouldn't affect anything, recursive sub modules doesn't pull anything extra